src/plots/ternary/index.js

```diff
@@ -50,7 +50,7 @@ exports.plot = function plotTernary(gd) {
                 fullLayout
             );
 
-            fullLayout[ternaryId]._ternary = ternary;
+            fullLayout[ternaryId]._subplot = ternary;
```

Comment:
One step to generalize how we store instances to subplot instance in `fullLayout`.

To note, I haven't changed all the `fullLayout.scene._scene` and `fullLayout.geo._geo` occurrences as these refs may have leaked in the wild. We should do so in `v2.0.0` though.